### PR TITLE
Detail views for Networks and Volumes + standard Layout

### LINKED ISSUE #149: Tasks
#### Display
- Name
- Image
- Node
- Status

#### Links
- View Task details `/tasks/{task_id}`

#### Display
- ID
- Service
- Name
- Created
- Last Update
- Image
- Node
- Status
  - State
  - Desired State

### LINKED ISSUE #167: Remove paging from datatables
Since we are already fetching all the data it would be more convenient to display everything at once.

### LINKED ISSUE #169: Detail view pages

### LINKED ISSUE #177: Use standard layout for all pages
Currently the following pages are not consistent with the layout:
- [ ] Users
- [ ] Networks
- [ ] Nodes

See the Service page for the preferred layout.

Layout suggestions are welcome

### LINKED ISSUE #179: Strip image metadata from Task list
Everything after the `@` can be discarded

### LINKED ISSUE #180: Display node name instead of ID in Task List
If this requires backend changes then an API spec should be drafted

### PR DESCRIPTION
**Pull request by ghost**
Closes #180
Closes #179
Closes #177 
Closes #169 
Closes #167 
Closes #149 

Requires TripleParity/docks-api/pull/81

Sorry that this took a little while but I this should be good to go. This branch did kind of touch a bit of everything that is why I would like to get it merged as soon as possible 

As stated the detail views for Volumes and Networks are done:
Networks
image
Volumes
image

Also a consistent layout has been made across the site, pages affected are the following:
Users:
image
And Nodes (I just aligned the search bar to the left as it is throughout)

NB! I have not worked on webhooks I think I would like  to make a separate pool request for weebhooks as @egeldenhuys  says there is a lot more layout there and I figured this pool request needs to take priority.

Also we have a new NavBar If you noticed in the previous screen shots navigating to a page displays what page you are on which is a nice feature.

Also the Task datatables got cleaned up closing issues #180 and #179

### CODE REVIEW

File: src/app/pages/tasks/list-view/task-list-view.component.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@ import { Task } from '../../../models/task/task.model';
 import { TaskService, TaskError } from '../../../services/task/task.service';
 import { ToastrService } from 'ngx-toastr';
 import { Router } from '@angular/router';
+import { NodeService, NodeError } from 'services/node/node.service';
+import { Node } from 'app/models/node/node.model';
+import { element } from 'protractor';
```
Comment (egeldenhuys) [inline src/app/pages/tasks/list-view/task-list-view.component.ts:8]: What is this?

File: src/app/pages/volumes/volume-detail-view/volume-detail-view.component.ts [hunk 2]
```diff
@@ -1,12 +1,56 @@
 import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute, ParamMap } from '@angular/router';
+import { VolumeService, VolumeError } from 'services/volume/volume.service';
+import { ToastrService } from 'ngx-toastr';
+import { Volume } from 'app/models/volume/volume.model';
+import { debug } from 'util';
```
Comment (egeldenhuys) [inline src/app/pages/volumes/volume-detail-view/volume-detail-view.component.ts:6]: What is this?

**Comment by egeldenhuys** [comment]
@Paulo-W Can you please do `npm run prettier` and commit?

**Review by annamarieHelberg — APPROVED** [review]
This looks great. Thank you @Paulo-W ! I think after the prettier it would be good to go.

**Review by egeldenhuys — APPROVED** [review]

**Pull request merged**